### PR TITLE
Pathways merge

### PR DESCRIPTION
**Pull request by matthewhirschey**
several bug fixes. Want to get this into ddh, so asking @pcastellanoescuder at this point to review and we'll get main up to date.

### CODE REVIEW

File: code/app/shiny_plots.R
Comment (pcastellanoescuder) [inline code/app/shiny_plots.R:1]: Why do we need to limit the structure plot to one protein when we have a panel to select which protein to visualize?

Captura de pantalla 2023-09-29 a las 11 10 36

**Review by pcastellanoescuder — APPROVED** [review]
I pushed one commit to unify the color scheme in the summary page (the protein card dash was blue instead of green) and tested the rest. Besides my comment on protein structure, the rest LGTM.

File: code/app/shiny_plots.R
Comment (pcastellanoescuder) [reply code/app/shiny_plots.R:1]: Already discussed with @matthewhirschey in Basecamp. Closing this.

**Review by pcastellanoescuder — APPROVED** [review]

**Comment by pcastellanoescuder** [comment]
@matthewhirschey can you review my last 2 commits? I'll be working on this branch for pathway-related stuff if that's ok.

**Review by matthewhirschey — COMMENTED** [review]
LGTM

**Pull request merged**